src/libraries/Microsoft.PowerFx.Core/Public/Types/FormulaType.cs

```diff
@@ -102,6 +102,25 @@ private static DType GetExpandedEntityType(DType expandEntityType, string relate
             return type;
         }
 
+        internal static FormulaType[] GetValidTypes()
```

Comment:
GetValidTypes is too general. We need to communicate the idea that these are the valid types for arguments to a UDF.

Reply:
Changed to include the word UDF